tests/functional/services/flows/test_flow_crud.py

```diff
@@ -23,6 +23,29 @@ def test_create_flow(flows_client, subscription_id):
         assert "subscription_id" not in req_body
 
 
+@pytest.mark.parametrize("value", [None, [], ["dummy_value"]])
+@pytest.mark.parametrize("key", ["run_managers", "run_monitors"])
+def test_create_flow_run_role_serialization(flows_client, key, value):
+    metadata = load_response(flows_client.create_flow).metadata
+    params = metadata["params"]
+
+    # Remove the key or set as empty list
+    if value is None:
+        params.pop(key)
+    else:
+        params[key] = value
```

Comment:
This still appears to be modifying the global dict. Based on comments, maybe a change was not pushed?